### DISCUSSION TITLE
Ensembl Gene IDs Displaying Instead of Gene Symbols

### DISCUSSION THREAD

**Discussion by mmccorkindale** [broadinstitute/seqr#3970] [Q&A] [answered]
Hi,

We're having some trouble when performing variant searches, namely that the gene IDs being returned are Ensembl:

![image](https://github.com/broadinstitute/seqr/assets/25675448/4ce65035-03cb-4215-a3ca-60afa05680f8)

In one of our earlier installs, gene symbol was the gene ID displayed in search results, which is more useful for our collaborators. Is this a case of changing a default parameter somewhere, or does this indicate that the gene symbols are not present in elasticsearch?

Thanks,

Michael

**Comment by hanars** [comment, marked as answer]
this occurs when the version of ensembl you have in the seqr database is incomplete. I would recommend running the follwoing to reset your reference data:
```
python manage.py update_all_reference_data --use-cached-omim
```
